### DISCUSSION TITLE
Indentation using 4 spaces

### DISCUSSION THREAD

**Discussion by Koushikphy** [fortran-lang/fortls#141] [Q&A] [answered]
How to configure the configuration file `.fortls` so that when using auto formatter it indents using 4 spaces ( instead of the default 3).

**Comment by gnikit** [comment]
`fortls` does not handle formatting, having said that what formatter are you using? I will convert this to a Q&A since it is not a bug or a feature req.

**Comment by Koushikphy** [comment, marked as answer]
I found out my VS Code is using `findent` by default. How do I choose between `findent` or `fprettify`. I found out `fprettify` uses an `.fprettify.rc` file for the configuration. But how do I set `fprettify` as default Fortran formatter for the modern Fortran extension.

**Reply by gnikit** [reply]
I think it would be a good idea for you to have a look at the README for the VS Code extension, specifically the formatting section: https://github.com/fortran-lang/vscode-fortran-support#formatting. You can specify arguments to the formatters without having to define a config file. It also shows how to change the formatter.

**Reply by Koushikphy** [reply]
I missed it somehow. Thanks.

**Reply by gnikit** [reply]
No worries, glad I could help.
